### PR TITLE
qemu.test: rework timedrift_adjust_time releated case

### PR DESCRIPTION
**Pull request by xutian**
Update:
    1. use epoch time to avoid timezone convert issue.
    2. use OS build-in shell to adjuest host/guest clock.
    3. use private brige to avoid guest sync time from network.
    4. remove s4 releated configuration.

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/timedrift_adjust_time.cfg [hunk 1]
```diff
@@ -53,34 +40,46 @@
...
+            time_difference = 3600
+        timedrift_adjust_time.guest_reboot.clock_host.adjust_guest_clock:
+            time_difference = 1800
+        timedrift_adjust_time.guest_reboot.clock_vm.adjust_guest_clock:
+            time_difference = 1800
+        timedrift_adjust_time.guest_reboot.clock_vm.adjust_host_clock:
+            time_difference = 1800
+        timedrift_adjust_time.guest_s3.clock_vm.adjust_host_clock:
+            time_difference = 1800
+    Win7:
```
Comment (spcui) [inline qemu/tests/cfg/timedrift_adjust_time.cfg]: Why only Win7 in here?

Comment (xutian) [inline qemu/tests/cfg/timedrift_adjust_time.cfg]: because Win7 have different behavior in my test.

**Comment by xutian** [comment]
@spcui 

thanks your comments,  has updated in latest commit.

**Comment by yaxue** [comment]
Run the related cases , all PASS.
LGTM
Acked-by: Yang Xue <yaxue@redhat.com>

**Comment by FengYang** [comment]
Thanks @xutian @yaxue @spcui , will merge it.

**Pull request merged**